packages/dialog/package.json

```
@@ -73,6 +73,7 @@
         "lit-html"
     ],
     "dependencies": {
+        "@spectrum-web-components/alert-dialog": "0.0.1",
```

```suggestion
        "@spectrum-web-components/alert-dialog": "^0.0.1",
```
That or it won't get managed correctly at release time.